backend/python/app/services/implementations/story_service.py

```diff
@@ -55,6 +55,23 @@ def create_story(self, story, content):
 
         return new_story
 
+    def get_stories_available_for_translation(self, user):
+        all_stories = []
+        approved_languages = user.approved_languages
+        for lang in approved_languages:
+            level = approved_languages[lang]
+            stories = (
+                Story.query.filter(Story.level <= level)
+                .filter(~Story.translated_languages.any(lang))
+                .all()
+            )
+
+            for story in stories:
```

Comment:
If the below `if `statement that I commented on works, I think we can skip this loop in favour of making `all_stories` a set and union-ing `stories` with it.
https://www.w3schools.com/python/ref_set_union.asp